Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Image/ImageSystemInterface.h

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include <Atom/RHI.Reflect/AttachmentId.h>
```

Comment:
Could this be forward declared?

Reply:
We can't since the AttachmentId was defined with using directive...

Reply:
I changed it to AZ::Name to keep it consistent with AttachmentImage::FindByUniqueName(const Name& uniqueAttachmentName) function